### PR TITLE
ft: clueso meld

### PR DESCRIPTION
**Pull request by JianqinWang**
To-do in docker-stack.yml:

Update the scality/s3server image tag
Update the management endpoint url

This is a rebased + squashed branch for the branch [lhs/clueso-meld](https://github.com/scality/Zenko/tree/lhs/clueso-meld) for merging into master

### CODE REVIEW

File: swarm-production/docker-stack.yml [hunk 1]
```diff
@@ -1,10 +1,8 @@
----
-
 version: "3.4"
 
 services:
   s3-data:
-    image: zenko/cloudserver:pensieve-0
+    image: scality/s3server:clueso-pensieve
```
Comment (LaurenSpiegel) [inline swarm-production/docker-stack.yml]: note: this will change to zenko/cloudserver:pensieve-1

File: swarm-production/docker-stack.yml [hunk 2]
```diff
@@ -94,6 +93,79 @@ services:
...
+    hostname: spark-master
+    depends_on:
+      - queue
+      - graphite
+      - lb
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    volumes:
+      - "heapdumps:/clueso/heapdumps:rw"
```
Comment (LaurenSpiegel) [inline swarm-production/docker-stack.yml]: @RachedBenMustapha, is keeping this in offensive?

File: swarm-production/secrets.txt [hunk 3]
```diff
@@ -1,2 +1,2 @@
-export SCALITY_ACCESS_KEY_ID=deployment-specific-access-key
-export SCALITY_SECRET_ACCESS_KEY=deployment-specific-secret-key
+export SCALITY_ACCESS_KEY_ID=VYFXV3UP05E583N1LL4F
```
Comment (LaurenSpiegel) [inline swarm-production/secrets.txt]: revert these changes

File: swarm-production/secrets.txt [hunk 4]
```diff
@@ -1,2 +1,2 @@
...
+export SCALITY_SECRET_ACCESS_KEY=30=Yw4dZg06mjGRYT2=l6JZvJVKN5yGg/muUKurW
```
Comment (LaurenSpiegel) [inline swarm-production/secrets.txt]: same

File: swarm-production/docker-stack.yml [hunk 5]
```diff
@@ -188,9 +260,11 @@ networks:
 volumes:
   s3-data:
   s3-metadata:
+  graphana-data:
```
Comment (RachedBenMustapha) [inline swarm-production/docker-stack.yml]: `grafana`?

File: swarm-production/docker-stack.yml (hunk 2 above)
Comment (RachedBenMustapha) [reply swarm-production/docker-stack.yml]: that's fine :)

File: swarm-production/docker-stack.yml [hunk 6]
```diff
@@ -52,7 +50,8 @@ services:
       METADATA_HOST: s3-metadata
       REDIS_HOST: cache
       ENDPOINT: "${ENDPOINT:-zenko}"
-      MANAGEMENT_ENDPOINT: 'https://api.zenko.io'
+      ## Temp change to dev endpoint for Clueso testing
+      MANAGEMENT_ENDPOINT: 'https://api-dev.private.zenko.io'
```
Comment (RachedBenMustapha) [inline swarm-production/docker-stack.yml]: this doesn't seem to be in a dropme commit

File: swarm-production/docker-stack.yml [hunk 7]
```diff
@@ -94,6 +93,79 @@ services:
         delay: "10s"
         monitor: "5s"
 
+  livy:
+    image: scality/clueso-livy:pensieve
+    hostname: livy
+    ports:
+      - "8998:8998"
```
Comment (RachedBenMustapha) [inline swarm-production/docker-stack.yml]: that's a lot of ports, which ones are really needed from the outside?

File: swarm-production/docker-stack.yml [hunk 8]
```diff
@@ -94,6 +93,79 @@ services:
...
+
+  spark-master:
+    image: scality/clueso-spark-master:pensieve
+    hostname: spark-master
+    depends_on:
+      - queue
+      - graphite
+      - lb
+    ports:
+      - "8080:8080"
```
Comment (RachedBenMustapha) [inline swarm-production/docker-stack.yml]: which of these ports need to be exposed to the outside?

File: swarm-production/docker-stack.yml (hunk 6 above)
Comment (JianqinWang) [reply swarm-production/docker-stack.yml]: I ended up squashing all the commits indiscriminately - I was going to just change the line later and squash again xD
